### ISSUE TITLE
Could not load NIB in bundle with name 'NSMainNibFile~ipad'

### ISSUE THREAD

**Issue by danielflippance** [ionic-team/ionic#12116]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[X ] **3.x**

**I'm submitting a ...**  (check one with "x")
[X ] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
When adding the ios platform, the resulting .plist file is missing strings for NSMainNibFile & NSMainNibFile~ipad

    $ Ionic cordova platform add ios

The result is missing <string/> tags in the .plist file. /platforms/ios/myapp.plist:

```
    <key>NSMainNibFile</key>
    <key>NSMainNibFile~ipad</key>
    <key>UISupportedInterfaceOrientations</key>
    ....
```

I expected this to look like:

```
    <key>NSMainNibFile</key>
    <string></string>
    <key>NSMainNibFile~ipad</key>
    <string></string>
    <key>UISupportedInterfaceOrientations</key>
    ....
```
Running the app results in the error:

```
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', 
reason: 'Could not load NIB in bundle: 'NSBundle 
</var/containers/Bundle/Application/78ABCDC1-ABCD-ABCD-9793-891421ABCDB8/myapp.app>
 (loaded)' with name 'NSMainNibFile~ipad''
```

**Steps to reproduce:**

    $ Ionic cordova platform add ios

**Related code:**

I tried adding the following to config.xml, but with the same outcome:
```
    <platform name="ios">
        ...
        <config-file parent="NSMainNibFile" platform="ios" target="*-Info.plist">
            <string/>
        </config-file>
        <config-file parent="NSMainNibFile~ipad" platform="ios" target="*-Info.plist">
            <string/>
        </config-file>
    </platform>
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
$ ionic info

global packages:

    @ionic/cli-utils : 1.4.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.4.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.4.0
    @ionic/cli-plugin-ionic-angular : 1.3.1
    Cordova Platforms               : ios 4.4.0
    Ionic Framework                 : ionic-angular 3.4.0

System:

    Node       : v6.10.0
    OS         : OS X El Capitan
    Xcode      : Xcode 8.2.1 Build version 8C1002 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.13 
    npm        : 3.10.10 
```

**Comment by danielflippance** [comment]
There was previously an issue with the same symptoms that was resolved by cordova 7.0.0, which I'm using - https://forum.ionicframework.com/t/xcode-main-interface-nsmainnibfile-ipad/66942/4

Plugins used:
```
    <plugin name="audiotoggler" spec="0.0.7"/>
    <plugin name="com.xmartlabs.cordova.market" spec="~1.1"/>
    <plugin name="cordova-plugin-actionsheet" spec="2.3.3"/>
    <plugin name="cordova-plugin-app-preferences" spec="0.99.3"/>
    <plugin name="cordova-plugin-app-version" spec="0.1.9"/>
    <plugin name="cordova-plugin-audiotoggler" spec="~0.0.6"/>
    <plugin name="cordova-plugin-camera" spec="2.4.1">
        <variable name="CAMERA_USAGE_DESCRIPTION" value=" "/>
        <variable name="PHOTOLIBRARY_USAGE_DESCRIPTION" value=" "/>
    </plugin>
    <plugin name="cordova-plugin-compat" spec="1.1.0"/>
    <plugin name="cordova-plugin-console" spec="1.0.7"/>
    <plugin name="cordova-plugin-device" spec="1.1.6"/>
    <plugin name="cordova-plugin-file" spec="4.3.3"/>
    <plugin name="cordova-plugin-file-transfer" spec="1.6.3"/>
    <plugin name="cordova-plugin-http" spec="1.2.0"/>
    <plugin name="cordova-plugin-inappbrowser" spec="1.7.1"/>
    <plugin name="cordova-plugin-keyboard" spec="1.1.5"/>
    <plugin name="cordova-plugin-market" spec="^1.1.3"/>
    <plugin name="cordova-plugin-media-capture" spec="1.4.3"/>
    <plugin name="cordova-plugin-network-information" spec="1.3.3"/>
    <plugin name="cordova-plugin-screen-orientation" spec="2.0.1"/>
    <plugin name="cordova-plugin-settings-hook" spec="0.2.5"/>
    <plugin name="cordova-plugin-splashscreen" spec="4.0.3"/>
    <plugin name="cordova-plugin-statusbar" spec="2.2.3"/>
    <plugin name="cordova-plugin-video-editor" spec="1.1.2"/>
    <plugin name="cordova-plugin-whitelist" spec="1.3.2"/>
    <plugin name="cordova-plugin-x-socialsharing" spec="5.1.8"/>
    <plugin name="cordova-plugin-x-toast" spec="2.6.0"/>
    <plugin name="ionic-plugin-keyboard" spec="2.2.1"/>
```

NPM Dependencies:
```
    "dependencies": {
        "@angular/common": "4.1.3",
        "@angular/compiler": "4.1.3",
        "@angular/compiler-cli": "4.1.3",
        "@angular/core": "4.1.3",
        "@angular/forms": "4.1.3",
        "@angular/http": "4.1.3",
        "@angular/platform-browser": "4.1.3",
        "@angular/platform-browser-dynamic": "4.1.3",
        "@angular/platform-server": "4.1.3",
        "@ionic-native/action-sheet": "3.12.1",
        "@ionic-native/app-preferences": "3.12.1",
        "@ionic-native/app-version": "3.12.1",
        "@ionic-native/camera": "3.12.1",
        "@ionic-native/core": "3.12.1",
        "@ionic-native/file": "3.12.1",
        "@ionic-native/http": "3.12.1",
        "@ionic-native/in-app-browser": "3.12.1",
        "@ionic-native/keyboard": "3.12.1",
        "@ionic-native/market": "3.12.1",
        "@ionic-native/media-capture": "3.12.1",
        "@ionic-native/network": "3.12.1",
        "@ionic-native/screen-orientation": "3.12.1",
        "@ionic-native/social-sharing": "3.12.1",
        "@ionic-native/splash-screen": "3.12.1",
        "@ionic-native/status-bar": "3.12.1",
        "@ionic-native/toast": "3.12.1",
        "@ionic-native/transfer": "3.12.1",
        "@ionic-native/video-editor": "3.12.1",
        "@ionic/storage": "2.0.1",
        "audiotoggler": "0.0.7",
        "com.xmartlabs.cordova.market": "~1.1",
        "compare-versions": "3.0.1",
        "cordova-ios": "^4.4.0",
        "cordova-plugin-actionsheet": "2.3.3",
        "cordova-plugin-app-preferences": "0.99.3",
        "cordova-plugin-app-version": "0.1.9",
        "cordova-plugin-audiotoggler": "~0.0.6",
        "cordova-plugin-camera": "2.4.1",
        "cordova-plugin-compat": "1.1.0",
        "cordova-plugin-console": "1.0.7",
        "cordova-plugin-device": "1.1.6",
        "cordova-plugin-file": "4.3.3",
        "cordova-plugin-file-transfer": "1.6.3",
        "cordova-plugin-http": "1.2.0",
        "cordova-plugin-inappbrowser": "1.7.1",
        "cordova-plugin-keyboard": "1.1.5",
        "cordova-plugin-market": "^1.1.3",
        "cordova-plugin-media-capture": "1.4.3",
        "cordova-plugin-network-information": "1.3.3",
        "cordova-plugin-screen-orientation": "2.0.1",
        "cordova-plugin-settings-hook": "0.2.5",
        "cordova-plugin-splashscreen": "4.0.3",
        "cordova-plugin-statusbar": "2.2.3",
        "cordova-plugin-video-editor": "1.1.2",
        "cordova-plugin-whitelist": "1.3.2",
        "cordova-plugin-x-socialsharing": "5.1.8",
        "cordova-plugin-x-toast": "2.6.0",
        "ionic-angular": "3.4.0",
        "ionic-native": "2.9.0",
        "ionic-plugin-keyboard": "2.2.1",
        "ionicons": "3.0.0",
        "portfinder": "1.0.13",
        "raven-js": "3.16.0",
        "reflect-metadata": "0.1.10",
        "remove": "0.1.5",
        "rxjs": "5.4.0",
        "setimmediate": "1.0.5",
        "sw-toolbox": "3.6.0",
        "zone.js": "0.8.12"
    },
```

**Comment by danielflippance** [comment]
I removed all Cordova plugins one by and the bug seems to be caused by `cordova-plugin-settings-hook`. Removing that plugin fixes the problem.

I have opened a bug on that repo: https://github.com/mircoc/cordova-plugin-settings-hook/issues/12
